demo1.py

```diff
@@ -0,0 +1,69 @@
+import os
+import csv
+from skingpt4.common.config import Config
+from skingpt4.common.registry import registry
+from skingpt4.conversation.conversation import Chat, CONV_VISION
+
+
+def initialize_chat(cfg_path, gpu_id):
+    """Initialize the Chat instance."""
+    print('Initializing Chat')
+    cfg = Config({"cfg_path": cfg_path})
+
+    model_config = cfg.model_cfg
+    model_config.device_8bit = gpu_id
+    model_cls = registry.get_model_class(model_config.arch)
+    model = model_cls.from_config(model_config).to('cuda:{}'.format(gpu_id))
+
+    vis_processor_cfg = cfg.datasets_cfg.cc_sbu_align.vis_processor.train
+    vis_processor = registry.get_processor_class(vis_processor_cfg.name).from_config(vis_processor_cfg)
+    chat = Chat(model, vis_processor, device='cuda:{}'.format(gpu_id))
+    print('Initialization Finished')
+    return chat
+
```

Comment:
I think we still need to setup the seeds 
```suggestion
def setup_seeds(config):
    seed = config.run_cfg.seed + get_rank()

    random.seed(seed)
    np.random.seed(seed)
    torch.manual_seed(seed)

    cudnn.benchmark = False
    cudnn.deterministic = True
```